library/include/ck/library/utility/literals.hpp

```diff
@@ -3,6 +3,7 @@
 
 #pragma once
 
+namespace literals {
```

Comment:
Since this is under ```include/ck```

maybe namespace ```ck::iternals``` make more sense?

Reply:
ok, I moved it to as `ck`'s sub-namespace